components/Layout/Layout.js

```diff
@@ -2,20 +2,49 @@ import Link from "next/link";
 import Header from "../Header/Header";
 import Image from "next/image";
 import { NavBar } from "../Layout/Layout.styled";
+import { useState } from "react";
 export default function Layout({ children }) {
+  const [activeLink, setActiveLink] = useState("");
```

Comment:
Please do not create a state mirroring the "state" that you already have in your route.

You could use the following instead:

```suggestion
  const { asPath } = useRouter();
```

This requires you to import `useRouter` from `"next/router"`;

Reply:
thank you for your feedback I`ll do change it now